rasa_nlu/server.py

```diff
@@ -9,6 +9,7 @@
 
 import simplejson
 import six
+import time
```

Comment:
why `import time`?

Reply:
this is an artefact of some local testing and should not be there 😄